baseledger/x/bridge/keeper/msg_server_orchestrator_validator_address.go

```diff
@@ -30,6 +30,14 @@ func (k msgServer) CreateOrchestratorValidatorAddress(goCtx context.Context, msg
 		return nil, sdkerrors.Wrap(sdkerrors.ErrNotFound, "validator not found")
```

Comment:
this is not the same check as the one bellow?

Reply:
which one? this is checking if validator exists